packages/abi-gen/test-cli/output/typescript/abi_gen_dummy.ts

```diff
@@ -19,42 +19,50 @@ import {
     SupportedProvider,
 } from 'ethereum-types';
 import { BigNumber, classUtils, logUtils, providerUtils } from '@0x/utils';
-import { EventCallback, IndexedFilterValues, SimpleContractArtifact } from '@0x/types';
+import { EventCallback, IndexedFilterValues, SimpleContractArtifact, TxOpts } from '@0x/types';
 import { Web3Wrapper } from '@0x/web3-wrapper';
 import { assert } from '@0x/assert';
 import * as ethers from 'ethers';
 // tslint:enable:no-unused-variable
 
-export type AbiGenDummyEventArgs = AbiGenDummyWithdrawalEventArgs | AbiGenDummySimpleEventEventArgs;
+export type AbiGenDummyEventArgs = AbiGenDummySimpleEventEventArgs | AbiGenDummyWithdrawalEventArgs;
```

Comment:
Is this ordering nondeterministic?

Reply:
It's deterministic but the latest version of sol-compiler uses `soljson-v0.5.12`, whereas the existing artifacts were compiled with `soljson-v0.5.10`. So I assume this has something to do with the ordering.

I actually found an existing method in the dummy contract that let me do what I needed, so I removed the contract changes and reverted the artifact. We should open a separate PR just to update the artifacts though so that future diffs will be clean. I'll open an issue and do it after this one is merged.